test/unit/event/send-acknowledgement-event.test.js

```diff
@@ -0,0 +1,56 @@
+const mockFRN = require('../../mock-frn')
+const sendAcknowledgementEvent = require('../../../app/event/send-acknowledgement-event')
+
+jest.mock('../../../app/event/send-processing-ack-error-event')
+const sendProcessingAckErrorEvent = require('../../../app/event/send-processing-ack-error-event')
+
+jest.mock('../../../app/event/send-invalid-bank-details-event')
+const sendInvalidBankDetailsEvent = require('../../../app/event/send-invalid-bank-details-event')
+
+let mockAcknowledgementError
+let mockHoldCategoryName
+
+describe('send acknowledgement event', () => {
+  beforeEach(() => {
+    mockAcknowledgementError = require('../../mock-acknowledgement-error')
```

Comment:
Going to need to assign this to `JSON.parse(JSON.stringify(require('../../mock-acknowledgement-error')))`

Reason being, when we change the content of this `mockAcknowledgementError` in 1 test (say to an invalid key/value to trigger so sort of validation failure) that change will persist for all following tests. We don't want that and want our tests and test cases to run in complete isolation of each other, not caring which order they're ran in.

Reply:
Would this need changed in every test file that I've made a similar reference to?

Reply:
Yup, all of our import of mock objects should all be imported in this way.